test/TimerTest.cpp

```diff
@@ -123,14 +124,14 @@ TEST(TimeoutTimer, Limited) {
   }
 }
 
-TEST(TimeBlockAndLock, TimeBlockAndLock) {
+TEST(TimeBlockAndLog, TimeBlockAndLog) {
   std::string s;
   {
     auto callback = [&s](size_t msecs, std::string_view message) {
       s = absl::StrCat(message, ": ", msecs);
     };
     ad_utility::TimeBlockAndLog t{"message", callback};
-    std::this_thread::sleep_for(10ms);
+    std::this_thread::sleep_for(25ms);
   }
-  ASSERT_EQ("message: 10", s);
+  ASSERT_THAT(s, ::testing::MatchesRegex("message: 2[0-9]"));
```

Comment:
To be honest I'd still consider printing anything less than 25ms a failure, so you might want to consider changing this
```suggestion
  ASSERT_THAT(s, ::testing::MatchesRegex("message: 2[5-9]"));
```